ToDoList/DTOs/ToDoUpdateCompleteDto.cs

```diff
@@ -4,7 +4,6 @@ namespace ToDoList.DTOs
 {
     public class ToDoUpdateCompleteDto
     {
-        [Required]
```

Comment:
this whole Dto model is not needed, just in the controller get item by id using the repository, and set completed = true, then save, do the same for text change, no need for these weird models, they add unnecessary complexity